Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -396,7 +396,11 @@ def uninstall_rmdir(*directories, command: nil, **_)
               command.run!("/bin/rm", args: ["-f", "--", ds_store], sudo: true, print_stderr: false)
             end
 
-            command.run("/bin/rmdir", args: ["--", resolved_path], sudo: true, print_stderr: false)
+            paths = Dir["#{resolved_path}/*"]
+            uninstall_rmdir(*paths, command: command) unless paths.select(&File.method(:file?)).any?
```

Comment:
Use `Pathname.glob` above and `paths.all?(&:directory?)` here.

Reply:
If we’re only globbing `*`, wouldn’t it be clearer to use `Pathname.children` instead?

Reply:
> Use `paths.all?(&:directory?)` here.

@reitermarkus Nice, I didn't know about `.all?` :)